ups/meas_extensions_photometryKron.cfg

```diff
@@ -4,11 +4,10 @@ import lsst.sconsUtils
 
 dependencies = {
     "required": ["utils", "afw", "meas_base"],
-    "buildRequired": ["boost_test", "swig"],
+    "buildRequired": ["pybind11"],
 }
 
 config = lsst.sconsUtils.Configuration(
     __file__,
     hasDoxygenInclude=False,
-    hasSwigFiles=True,
```

Comment:
That was discussed recently and it is OK. Yes the default is True, but it makes no difference and I'd rather eliminate all reference to SWIG. sconsUtils will get an update at some point to either eliminate this argument or at least change the default to false.